content/issues/2023-07-03-testing-provisioning-failed.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Guest couldn't be provisioned: Artemis resource ended in 'error' state"
+date: "2023-07-03T08:00:00+02:00"
+affected:
+  - Testing Farm
+resolved: false
+section: issue
+severity: notice
+---
+
+Some jobs submitted by Packit to the Testing Farm are pending with this error:
+`"Guest couldn't be provisioned: Artemis resource ended in 'error' state"`.
+
+Testing Farm has already resolved the issue; please re-submit the job with a
+comment like `/packit test` or through the GitHub checks button.
```

Comment:
What about linking something like this?
https://packit.dev/docs/configuration/upstream/tests/#running-tests-with-a-specific-identifier

(So we market these features a bit..;)